server/API/REST.php

```diff
@@ -441,21 +441,22 @@ function book_lecture($vars) {
 			$stmt->bindValue(':userId', $userId, PDO::PARAM_INT);
 			$stmt->bindValue(':student', USER_TYPE_STUDENT, PDO::PARAM_INT);
 			if (!$stmt->execute()) {
-				throw new PDOException($stmt->errorInfo()[2], $stmt->errorInfo()[0]);
+				throw new PDOException($stmt->errorInfo()[2]);
 			}
-			if (!$stmt->fetch()) {
+			if (!$user_data = $stmt->fetch()) {
```

Comment:
Make it more clear
```suggestion
			$user_data = $stmt->fetch();
			if (!$user_data) {
```

Reply:
that syntax is widely used. If you insist i commit the change, anyway